app/views/proposals/_tooltip.html.haml

```diff
@@ -1,11 +1,12 @@
 - progress_status = event.progress_status
 %ul.list-unstyled
-  %li{'class'=>class_for_todo(progress_status['registered'])}
-    %span{'class'=>icon_for_todo(progress_status['registered'])}
-      - if progress_status['registered']
-        Speaker(s) registered to the conference
-      - else
-        = link_to 'Speaker(s) not registered to the conference', new_conference_conference_registration_path(event.program.conference.short_title)
+  - if @conference.registration_open? || (@conference.user_registered? current_user)
```

Comment:
I was thinking that there are four cases for this line https://github.com/namangupta01/osem/blob/master/app/views/proposals/_tooltip.html.haml#L9 
These are :

1. If registrations are not open and speaker haven't registered, then there should not be any link for registration in complete your proposal box. And also, we should not count the registration  **process in progress bar**. which is what i am trying to do in else block https://github.com/namangupta01/osem/blob/master/app/views/proposals/index.html.haml#L83

2.If registrations are not open and speaker have registered, then we can show them that they have already registered (which is what i am thinking of doing, which leads to not using of "user can register" statement )

3.If registrations are open and speaker haven't registered, then it should show speaker to do registration.

4.If registration are open and speaker have registered, then it should show that user have registered.

Note : When User have registered for the conference, the registration link would turn into green and also the registration link is **no more a link, its just a text showing them** that they have successfully registered.

If we follow above four statements, then we cannot use the ability of user to create new registration.
What do you think of this?